docs/manuals/user/dataMigrationFromSth-cometToQL/application.properties

```diff
@@ -0,0 +1,17 @@
+#settings

```

Comment:
are there default values for any of these that could make sense? (e.g., could qlPort be left as 8668 by default?) The more sensible defaults you put the easier it is for people to use